Hakawai/Mentions/HKWMentionsPlugin.m

```diff
@@ -1606,6 +1606,16 @@ - (BOOL)textViewShouldChangeTextInRangeV2:(NSRange)range
     if ([text length] == 0 && range.length == 1) {
```

Comment:
update comment above:

// In simple refactor, we focus on deletions in order to allow for personalization/deletions of mentions, and insertions to hand mention highlighting

Reply:
Updated code comment.